### PR TITLE
chore: add setup files

### PR DESCRIPTION
**Pull request by jsmassa**
Setup files copied from the original [heissepreise](https://github.com/badlogic/heissepreise/tree/main/docker) with only a few adjustments.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+FROM node:19
+
+WORKDIR /heisse-preise-de/
```
Comment (TimoKramer) [inline Dockerfile:3]: You could put the entrypoint in here and a default cmd as well
```suggestion
WORKDIR /heisse-preise-de/

EXPOSE 3000

LABEL org.opencontainers.image.source=https://github.com/yobst/heisse-preise-de
LABEL org.opencontainers.image.description="Yobst Price Scraper"

ENTRYPOINT npm
CMD ["run", "start"]
```

File: README.md [hunk 2]
```diff
@@ -51,19 +51,32 @@ Once the app is listening per default on port 3000, open <http://localhost:3000>
...
-npm run start
+docker build . -t localhost/heisse-preise-de_site:latest
+```
+
+#### Deployment
+
+For the _development_ version run following command:
+
+```
+docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data --publish 3000:3000 --entrypoint npm localhost/heisse-preise-de_site:latest run dev
```
Comment (TimoKramer) [inline README.md:69]:
```suggestion
docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data --publish 3000:3000  localhost/heisse-preise-de_site:latest run dev
```

File: README.md [hunk 3]
```diff
@@ -51,19 +51,32 @@ Once the app is listening per default on port 3000, open <http://localhost:3000>
...
 ```
 
 Once the app is listening per default on port 3000, open <http://localhost:3000> in your browser.
 
+For the _production_ environment run the following:
+
+```
+docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data --publish 3000:3000 --entrypoint npm localhost/heisse-preise-de_site:latest run start
```
Comment (TimoKramer) [inline README.md:77]:
```suggestion
docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data --publish 3000:3000 localhost/heisse-preise-de_site:latest
```

**Review by TimoKramer — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (jsmassa) [reply Dockerfile:3]: Changed it accordingly

File: README.md (hunk 2 above)
Comment (jsmassa) [reply README.md]: I have updated the commands.

File: README.md (hunk 3 above)
Comment (jsmassa) [reply README.md]: Changed.

File: README.md [hunk 4]
```diff
@@ -51,19 +51,32 @@ Once the app is listening per default on port 3000, open <http://localhost:3000>
...
+```bash
+docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data localhost/heisse-preise-de_site:latest run dev
 ```
 
 Once the app is listening per default on port 3000, open <http://localhost:3000> in your browser.
 
+For the _production_ environment run the following:
+
+```bash
+docker run --volume=$(pwd):/heisse-preise-de --volume=$(pwd)/data/:/heisse-preise-de/data localhost/heisse-preise-de_site:latest run start
```
Comment (TimoKramer) [inline README.md:77]: run start is now the default and is not needed here anymore

**Review by TimoKramer — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (jsmassa) [reply README.md:77]: I know, I only left it in so the difference is obvious. But I can add the alternative.

I found an issue with the `npm run start` though as the scripts have changed in this version of the project and `start` has been dropped - in favor of a [separate build](https://github.com/yobst/heisse-preise-de/blob/7d859895662ad0136ce6359f181c58a75a56e629/package.json#L15) process if I understand it correctly. 

I would add `npm run build` to the dockerfile then and set `npm run server` as the default command. Or what do you think?

Comment (jsmassa) [reply README.md]: I have added a build stage now to the Dockerfile. 

I also dropped the documentation about running dev mode inside the container. Shouldn't be needed anyway. The documented command for production and staging is now shortened and corrected.

**Review by TimoKramer — APPROVED** [review]

**Pull request merged**